addons/portal_rating/static/src/chatter/frontend/message_patch.xml

```diff
@@ -2,38 +2,38 @@
 <templates xml:space="preserve">
     <t t-inherit="mail.Message" t-inherit-mode="extension">
         <xpath expr="//div[@name='header']" position="after">
-            <div t-if="ratingValue" class="mb-2">
+            <div t-if="message.rating_id?.rating" class="mb-2">
                 <t t-call="portal_rating.rating_stars_static">
-                    <t t-set="val" t-value="ratingValue"/>
+                    <t t-set="val" t-value="message.rating_id.rating"/>
                 </t>
             </div>
         </xpath>
         <xpath expr="//*[@t-ref='messageContent']" position="inside">
             <!--Only possible if a rating is link to the message, for now we can't comment if no rating
                 is link to the message (because publisher comment data
                 is on the rating.rating model - one comment max) -->
-            <div t-if="message.rating and message.rating.id" class="o_wrating_publisher_container">
+            <div t-if="message.rating_id" class="o_wrating_publisher_container">
                 <button t-if="store.self.is_user_publisher"
-                    t-attf-class="btn px-2 my-2 btn-sm border o_wrating_js_publisher_comment_btn {{ message.rating.publisher_comment !== '' ? 'd-none' : '' }}"
-                    t-att-data-mes_index="message.rating.mes_index"
+                    t-attf-class="btn px-2 my-2 btn-sm border o_wrating_js_publisher_comment_btn {{ message.rating_id.publisher_comment ? 'd-none' : '' }}"
```

Comment:
This rename is a bit noisy, could we maybe rename rating_id of the old formatter in a prep PR?